src/modules/auth/auth.routes.ts

```diff
@@ -63,14 +64,14 @@ export const registerRoute = createRoute({
           password: z.string().min(8).max(60),
         }),
       ),
-      "The validation error(s)",
+      VALIDATION_ERROR_MESSAGE,
     ),
     [HTTPStatusCodes.INTERNAL_SERVER_ERROR]: jsonContent(
       z.object({
         success: z.boolean().default(false),
         message: z.string(),
       }),
-      "Server side error(s)",
+      SERVER_ERROR,
```

Comment:
For this we should have INTERNAL_SERVER_ERROR which should say 'Something went wrong, please try again.'.